src/mavsdk/core/mavlink_mission_transfer_server.cpp

```diff
@@ -301,4 +333,363 @@ void MavlinkMissionTransferServer::ReceiveIncomingMission::callback_and_reset(Re
     _done = true;
 }
 
+MavlinkMissionTransferServer::SendOutgoingMission::SendOutgoingMission(
```

Comment:
The fast majority of this block of changes was copied from `MavlinkMissionTransferClient::UploadWorkItem` with minor adjustments.